stac_mjx/stac_core.py

```
@@ -10,8 +10,8 @@
 
 import optax
 
-from stac_mjx import operations as op
-from stac_mjx import utils
+from stac_mjx import op_utils
+from stac_mjx import io
```

Ruff

`stac_mjx.io` imported but unused

Remove unused import: `stac_mjx.io`

(F401)